src/models/disassemblyoutput.cpp

```diff
@@ -308,22 +308,26 @@ DisassemblyOutput DisassemblyOutput::disassemble(const QString& objdump, const Q
     }
     arguments.append(binary);
 
-    asmProcess.start(objdump, arguments);
+    asmProcess.start(processPath, arguments);
 
     if (!asmProcess.waitForStarted()) {
-        disassemblyOutput.errorMessage += QApplication::translate("DisassemblyOutput", "Process was not started.");
+        disassemblyOutput.errorMessage += QApplication::translate("DisassemblyOutput", "Process was not started.")
```

Comment:
string puzzle, this is impossible to translate properly. Make this one large string and then use arg to fill in the placeholders and also include the `QProcess::errorString`

```
..., "<qt>Process failed to start: <tt>%1 %2</tt> returned <tt>%2</tt>").arg(processPath, arguments.join(...), asmProcess.errorString())
```